### PR TITLE
Update getDelimitedInput to remove empty values.

### PR DESCRIPTION
**Pull request by ericsciple**

### CODE REVIEW

File: node/test/tasklib.ts [hunk 1]
```diff
@@ -564,6 +596,32 @@ describe('Test vsts-task-lib', function() {
 
             done();
         })
+
+        // filePathSupplied tests
```
Comment (ericsciple) [inline node/test/tasklib.ts]: I just moved these two tests. I was trying to figure out what was covered already.

The tests were largely grouped already by the function being tested. Only these two were out of place.

Comment (bryanmacfarlane) [inline node/test/tasklib.ts]: OK - I have a note to move into separate files/suites as well

**Comment by bryanmacfarlane** [comment]
Need to bump version in package.json

**Comment by bryanmacfarlane** [comment]
Thanks

**Comment by bryanmacfarlane** [comment]
Published to npm.

**Pull request merged**